data/json/vehicleparts/rams.json

```diff
@@ -38,7 +38,7 @@
     "damage_reduction": { "all": 70 }
   },
   {
-    "id": "ram_hardsteel",
+    "id": "ram_qt_steel",
```

Comment:
Just changing an item `id` just 'cause is a pretty bad idea
Lots of migration issues, yada yada, only change it if you have _really good_ reasons

Reply:
Oh yep this got caught in the find-replace.